### DISCUSSION TITLE
CONTRIBUTING.md corrections

### DISCUSSION THREAD

**Discussion by DevLab2425** [ProjectEvergreen/greenwood#1168] [General]
Hello Greenwoodians!

I've come to make some contributions to your fine product, however, I've encountered a few discrepancies in the guide that have introduced some struggles. I'd like to submit an issue and subsequent PR to help update the details to ensure their accuracy. 

The initial findings within [.github/CONTRIBUTING.md](https://github.com/ProjectEvergreen/greenwood/blob/0ff955495e0d62f13d0ec074547c6c864279832b/.github/CONTRIBUTING.md) are:

- Within the **Setup** section, the `nvm use` command (and optional Node installation) is needed before running `yarn install`; however, the current ordering is inverted. 
- Within the **Project Management / Discussions** section, the `time and considerations` link leads to a dead link marketing page (`https://www.jason.af/setting-expectations`).

**Comment by thescientist13** [comment]
Ah, that's a bummer about the link, and thanks for calling out the `nvm` issue.

Happy to see that put in the correct order, but it has been a while since that doc was put together.  If you don't mind kicking off a PR with the `nvm` steps updated, I'll try and figure out what that link should point to, and also refresh on any of the content in there.

Let me know your thoughts and if there's anything I can help answer re: contributing in the mean time.

**Comment by thescientist13** [comment]
@DevLab2425 
Just a heads up, but you _should_ have read / write access (except to protected branches) in this organization, so you shouldn't have to fork any of the repos to contribute, you _should_ just be able to just push your branches directly.

Let me know if that is not the case, and I will make that settings change.

**Comment by thescientist13** [comment]
Also wanted to extend an invite to our weekly community meetings ( @aholtzman and @hutchgrant are some friendly faces you may recognize 👋 ).

We meet (generally) every Wed @ 7pm EST.  Just keep an eye out in the **#greenwood** channel our Slack, as I drop a Google doc link with a Google Meets link for everyone to join from.  Would be great to have you there (but no pressure!)

**Comment by thescientist13** [comment]
Should all be updated now!  Thanks!
